config/quality/pmd/pmd-ruleset.xml

```diff
@@ -2,7 +2,8 @@
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Android Application Rules"
     xmlns="http://pmd.sf.net/ruleset/1.0.0"
     xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
-    xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd">
+    xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+    http://pmd.sf.net/ruleset_xml_schema.xsd">
 
```

Comment:
Two lines are required ?

Reply:
This solution is suggested by Gradle because this file has some issues and this line solves it.